activesupport/lib/active_support/callable.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  # An object that has case-equality to a Proc or Lambda by responding to #call.
+  # This can be used to duck-type match in a case statement.
+  module Callable
+    # Whether the object responds to #call
+    def self.===(other)
+      other.respond_to?(:call)
+    end
+  end
```

Comment:
```suggestion
  # Whether the object responds to #call
  Callable = ->(obj) { obj.respond_to?(:call) }
```

Reply:
Since there is no point including it. Might as well not make it a module at all.